script.js

```diff
@@ -1,4 +1,141 @@
+// BEAT THAT
+// Game mode (mode 1. play dice rolls) (mode 2. decide the dice order) (mode 3. decide the winner)
+// there are two players, two dices
+
+// global variables
+// randomly selected dice for each players
+var player1Dice1 = "";
+var player1Dice2 = "";
+var player2Dice1 = "";
+var player2Dice2 = "";
+
+// the numbers that decided by players
+var player1Num = "";
+var player2Num = "";
+
+//player score
+var player1Score = [];
+var player2Score = [];
+
+// game mode starts with player 1
+var gameMode = "player 1";
+
+// To get random number from dice and cover it to string
+var diceRoll = function () {
+  var randomNum = Math.floor(Math.random() * 6 + 1);
+  return randomNum.toString();
```

Comment:
[Suggestion] Is there really a need to change this to string? Since you want to store this as a number anyway.